src/helper/filterFunctions.js

```diff
@@ -83,35 +86,37 @@ export const sortBySortActive = (data, sortActive, sortDirection) => {
   }
 };
 
-export const handleFilterActive = (filterInputs, setFilterActive) => {
+export const handleFilterActive = filterInputs => {
+  const compareObjWith = { ...filterInputs };
   const emptyInputs = {
-    searchEmployee: '',
+    department: [],
+    month: [],
+    officeLocation: [],
     position: [],
-    cc: [],
-    cst: [],
-    officeLocation: []
-  };
-  if (JSON.stringify(emptyInputs) === JSON.stringify(filterInputs)) {
-    setFilterActive(false);
-  } else {
-    setFilterActive(true);
-  }
-};
-
-export const handleFormerFilterActive = (filterInputs, setFilterActive) => {
-  const emptyInputs = {
+    scStatus: [],
     searchEmployee: '',
-    year: [],
+    year: []
+  };
+  const emptyInputsFormerEmployee = {
+    department: [],
     month: [],
+    officeLocation: [],
     position: [],
-    cc: [],
-    cst: [],
-    officeLocation: []
+    searchEmployee: '',
+    year: []
   };
-  if (JSON.stringify(emptyInputs) === JSON.stringify(filterInputs)) {
-    setFilterActive(false);
+  if ('scStatus' in filterInputs) {
```

Comment:
I think this can be simplified into 4 lines:

```
if ('scStatus' in filterInputs) {
 return isEqual(compareObjWith, emptyInputs);
}
return isEqual(compareObjWith, emptyInputsFormerEmployee);

```

Reply:
You're right, just small change is needed - `return !isEqual(compareObjWith, emptyInputs);`..